src/main/java/uk/gov/hmcts/reform/professionalapi/configuration/LaunchDarklyConfiguration.java

```diff
@@ -29,6 +29,9 @@ public LDClient ldClient(@Value("${launchdarkly.sdk.key}") String sdkKey) {
      */
     @Override
     public void addInterceptors(InterceptorRegistry registry) {
-       // add entry
+        registry.addInterceptor(featureConditionEvaluation)
+            .addPathPatterns("/refdata/external/v1/organisations/mfa");
+        registry.addInterceptor(featureConditionEvaluation)
+            .addPathPatterns("/refdata/internal/v1/organisations/{orgId}/mfa");
```

Comment:
I may be wrong, but will it match for `{orgId}` and not an actual Organisation ID? Please test this before merging